plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3/S3SecurityMappingConfig.java

```diff
@@ -30,7 +31,7 @@
     private Duration refreshPeriod;
     private String colonReplacement;
 
-    public Optional<String> getConfigFilePath()
+    public Optional<@FileExists String> getConfigFilePath()
```

Comment:
The path might be http(s). Does `@FileExists` annotation work even for such protocols?
https://github.com/airlift/airlift/blob/master/configuration/src/main/java/io/airlift/configuration/validation/FileExistsValidator.java

Reply:
I think it won't work for http url. Can we revert this change for this class.

Reply:
Good catch. Thank you for spotting this one.
Are there multiple situations like this one which would warrant a `@ResourceExists` annotation / extension of the `FileExistsValidator`?

Reply:
we won't do any HTTP calls during config validation